results/bchoiced__RELLE/093417798a062ed8ddd8424df3be62145d4ace91/model_meta.json

```
@@ -0,0 +1,26 @@
+{
+    "name": "bchoiced/RELLE",
+    "revision": "093417798a062ed8ddd8424df3be62145d4ace91",
+    "release_date": "2025-04-17",
+    "languages": [],
+    "n_parameters": 7110672384,
+    "max_tokens": 32768.0,
+    "embed_dim": 4096,
+    "license": "CC-BY-SA-4.0 license",
```

Could you update the `model_meta` with the current information?
```suggestion
    "license": "cc-by-sa-4.0",
```